src/scripts/app.ts

```diff
@@ -1360,9 +1362,9 @@ export class ComfyApp {
           this.graph.change()
         }
 
-        if (e.key === '-' && e.altKey) {
+        if ((key === 'NumpadSubtract' || key === 'Minus') && e.altKey) {
           block_default = true
-          let scale = (this.ds.scale * 1) / 1.1
```

Comment:
Reverting the change this line as it is not related to what this PR is trying to achieve.